daily_table.js

```diff
@@ -241,311 +241,319 @@ const daily_table = [
-    { date: '2021-02-13', merged: 36.26, pr: 41, cxx20: 12, cxx23: 4, lwg: 2, issue: 322, bug: 104, avg_age: 103.80, avg_wait: 81.21, sum_age: 141.86, sum_wait: 110.99, },
-    { date: '2021-02-14', merged: 35.59, pr: 41, cxx20: 12, cxx23: 4, lwg: 2, issue: 322, bug: 104, avg_age: 104.80, avg_wait: 82.21, sum_age: 143.22, sum_wait: 112.35, },
-    { date: '2021-02-15', merged: 34.79, pr: 42, cxx20: 12, cxx23: 4, lwg: 2, issue: 322, bug: 104, avg_age: 103.29, avg_wait: 81.24, sum_age: 144.60, sum_wait: 113.73, },
-    { date: '2021-02-16', merged: 33.91, pr: 42, cxx20: 12, cxx23: 4, lwg: 2, issue: 323, bug: 104, avg_age: 104.29, avg_wait: 80.11, sum_age: 146.00, sum_wait: 112.16, },
-    { date: '2021-02-17', merged: 40.06, pr: 37, cxx20: 10, cxx23: 4, lwg: 2, issue: 324, bug: 104, avg_age: 114.11, avg_wait: 91.06, sum_age: 140.73, sum_wait: 112.31, },
-    { date: '2021-02-18', merged: 43.21, pr: 35, cxx20: 10, cxx23: 4, lwg: 2, issue: 325, bug: 105, avg_age: 120.54, avg_wait: 96.26, sum_age: 140.64, sum_wait: 112.30, },
-    { date: '2021-02-19', merged: 46.35, pr: 35, cxx20: 9, cxx23: 4, lwg: 2, issue: 325, bug: 105, avg_age: 120.39, avg_wait: 96.72, sum_age: 140.45, sum_wait: 112.84, },
-    { date: '2021-02-20', merged: 45.45, pr: 38, cxx20: 9, cxx23: 4, lwg: 2, issue: 325, bug: 105, avg_age: 111.85, avg_wait: 90.01, sum_age: 141.68, sum_wait: 114.01, },
-    { date: '2021-02-21', merged: 44.52, pr: 38, cxx20: 9, cxx23: 4, lwg: 2, issue: 325, bug: 105, avg_age: 99.80, avg_wait: 77.96, sum_age: 126.42, sum_wait: 98.74, },
-    { date: '2021-02-22', merged: 48.43, pr: 37, cxx20: 9, cxx23: 6, lwg: 6, issue: 330, bug: 106, avg_age: 98.22, avg_wait: 77.72, sum_age: 121.14, sum_wait: 95.86, },
-    { date: '2021-02-23', merged: 49.22, pr: 40, cxx20: 8, cxx23: 6, lwg: 6, issue: 332, bug: 106, avg_age: 90.75, avg_wait: 71.67, sum_age: 121.00, sum_wait: 95.56, },
-    { date: '2021-02-24', merged: 50.21, pr: 41, cxx20: 7, cxx23: 6, lwg: 6, issue: 332, bug: 106, avg_age: 89.42, avg_wait: 70.82, sum_age: 122.20, sum_wait: 96.79, },
-    { date: '2021-02-25', merged: 60.20, pr: 31, cxx20: 5, cxx23: 6, lwg: 6, issue: 330, bug: 105, avg_age: 118.11, avg_wait: 93.67, sum_age: 122.04, sum_wait: 96.79, },
-    { date: '2021-02-26', merged: 60.15, pr: 31, cxx20: 4, cxx23: 6, lwg: 6, issue: 330, bug: 105, avg_age: 116.16, avg_wait: 94.54, sum_age: 120.04, sum_wait: 97.69, },
-    { date: '2021-02-27', merged: 59.10, pr: 31, cxx20: 4, cxx23: 6, lwg: 6, issue: 330, bug: 105, avg_age: 117.16, avg_wait: 95.54, sum_age: 121.07, sum_wait: 98.73, },
-    { date: '2021-02-28', merged: 58.06, pr: 32, cxx20: 4, cxx23: 6, lwg: 6, issue: 331, bug: 105, avg_age: 114.48, avg_wait: 93.53, sum_age: 122.11, sum_wait: 99.77, },
+    { date: '2020-04-29', merged: 37.54, pr: 28, cxx20: 33, lwg: 5, issue: 184, bug: 44, avg_age: 48.05, avg_wait: 37.14, sum_age: 44.84, sum_wait: 34.66, },
```

Comment:
Thanks, I should have called this out like I did for the previous Status Chart update #1701. This happened because I dismissed your Apr 29, 2020 review of #664 in order to merge that PR. GitHub handles this retroactively, and querying GraphQL no longer shows that you Requested Changes - which affects the Sum and Avg Waits. So, this is expected.

Reply:
Great - I just wanted to be sure nothing funny was going on.